### PR TITLE
added filtres to users find

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by borovHokage — APPROVED** [review]

File: src/UserService.Data/Interfaces/IUserRepository.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public interface IUserRepository
 
         List<Guid> AreExistingIds(List<Guid> userIds);
 
-        List<DbUser> Find(int skipCount, int takeCount, out int totalCount);
+        (List<DbUser> dbUsers, int totalCount) Find(int skipCount, int takeCount, bool includeDeactivated = false);
```
Comment (BelehovEgor) [inline src/UserService.Data/Interfaces/IUserRepository.cs:25]: why you don't use filter here

**Review by ndoriomedov — APPROVED** [review]

File: src/UserService.Business/Commands/User/FindUserCommand.cs [hunk 2]
```diff
@@ -198,60 +215,88 @@ private IGetDepartmentUsersResponse GetUserIdsByDepartment(Guid departmentId, in
...
             dbUser,
-            _departmentInfoMapper.Map(getCompanyEmployeesResponse?.Departments?.FirstOrDefault(x => x.UsersIds.Contains(dbUser.Id))),
-            _positionInfoMapper.Map(getCompanyEmployeesResponse?.Positions?.FirstOrDefault(x => x.UsersIds.Contains(dbUser.Id))),
-            _imageInfoMapper.Map(images?.FirstOrDefault(x => x.ImageId == dbUser.AvatarFileId)),
-            _roleInfoMapper.Map(roles?.FirstOrDefault(x => x.UserIds.Contains(dbUser.Id))),
-            _officeInfoMapper.Map(getCompanyEmployeesResponse?.Offices?.FirstOrDefault(x => x.UsersIds.Contains(dbUser.Id))))));
-
-      result.TotalCount = totalCount;
-      result.Status = result.Errors.Any()
+
```
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/FindUserCommand.cs:279]: remove empty lines

**Review by BelehovEgor — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/UserService.Data/Interfaces/IUserRepository.cs (hunk 1 above)
Comment (MariyaSin) [reply src/UserService.Data/Interfaces/IUserRepository.cs]: done

File: src/UserService.Business/Commands/User/FindUserCommand.cs (hunk 2 above)
Comment (MariyaSin) [reply src/UserService.Business/Commands/User/FindUserCommand.cs]: done

**Pull request merged**